main.nf

```diff
@@ -3276,75 +3230,83 @@ def cnvkit_panel_version(task) {
-		//set group, id, file(mantaV), file(dellyV), file(melt), file(cnvkitV) \
-		//	from called_manta_panel.join(called_delly_panel, by:[0,1]).join(melt_vcf, by:[0,1]).join(called_cnvkit_panel, by:[0,1])
-		set group, id, file(vcfs), id, file(melt) from called_manta_panel.mix(called_delly_panel,called_cnvkit_panel,merged_gatk_panel).groupTuple().join(melt_vcf)
+		set group, id, file(vcfs) from called_manta_panel.mix(called_cnvkit_panel,merged_gatk_panel).groupTuple()
 				
 	output:
-		set group, id, file("${group}.merged.filtered.melt.vcf") into vep_sv_panel, annotsv_panel 
-		//set group, id, file("${group}.merged.filtered.vcf") into annotsv_panel
-		set group, file("${group}.merged.filtered.melt.vcf") into loqusdb_sv_panel
+		set group, id, file("${group}.merged.vcf") into ch_postprocess_merged_panel_sv
 		set group, file("*versions.yml") into ch_svdb_merge_panel_versions
 
 	script:
-		//tmp = mantaV.collect {it + ':manta ' } + dellyV.collect {it + ':delly ' } + cnvkitV.collect {it + ':cnvkit ' }
-		//vcfs = tmp.join(' ')
 		if (vcfs.size() > 1) {
 			// for each sv-caller add idx, find vcf and find priority, add in priority order! //
 			// index of vcfs added from mix //
 			manta_idx = vcfs.findIndexOf{ it =~ 'manta' }
-			delly_idx = vcfs.findIndexOf{ it =~ 'delly' }
 			cnvkit_idx = vcfs.findIndexOf{ it =~ 'cnvkit' }
 			gatk_idx = vcfs.findIndexOf{ it =~ 'gatk' }
 
 			// find vcfs //
 			manta = manta_idx >= 0 ? vcfs[manta_idx].collect {it + ':manta ' } : null
-			delly = delly_idx >= 0 ? vcfs[delly_idx].collect {it + ':delly ' } : null
 			cnvkit = cnvkit_idx >= 0 ? vcfs[cnvkit_idx].collect {it + ':cnvkit ' } : null
 			gatk = gatk_idx >= 0 ? vcfs[gatk_idx].collect {it + ':gatk ' } : null
-			tmp = manta + delly + gatk + cnvkit
+			tmp = manta + gatk + cnvkit
 			tmp = tmp - null
 			vcfs_svdb = tmp.join(' ')
 
 			// find priorities //
 			mantap = manta_idx >= 0 ? 'manta' : null
-			dellyp = delly_idx >= 0 ? 'delly' : null
 			gatkp = gatk_idx >= 0 ? 'gatk' : null
 			cnvkitp = cnvkit_idx >= 0 ? 'cnvkit' : null
-			tmpp = [mantap, dellyp, gatkp, cnvkitp]
+			tmpp = [mantap, gatkp, cnvkitp]
 			tmpp = tmpp - null
 			priority = tmpp.join(',')
 			
 			"""
-			source activate py3-env
-			svdb --merge --vcf $vcfs_svdb --no_intra --pass_only --bnd_distance 2500 --overlap 0.7 --priority $priority > ${group}.merged.vcf
-			filter_panel_cnv.pl --mergedvcf ${group}.merged.vcf --callers $priority > ${group}.merged.filtered.vcf
-			vcf-concat ${group}.merged.filtered.vcf $melt | vcf-sort -c > ${group}.merged.filtered.melt.vcf
-			
+			svdb \\
+			  --merge \\
+			  --vcf $vcfs_svdb \\
+			  --no_intra \\
+			  --pass_only \\
+			  --bnd_distance 2500 \\
+			  --overlap 0.7 \\
+			  --priority $priority \\
+			  --ins_distance 0 > ${group}.merged.tmp
+
+
+			# copy callers out of INFO.set to INFO.SCOUT_CUSTOM
+			add_callers_to_scout_custom.py \\
+				--callers $priority \\
+				--merged_vcf ${group}.merged.tmp > ${group}.merged.callers.tmp
```

Comment:
I'll change the .tmp suffix to .vcf and make the process publish spec more specific.

Realized now that you might want want to catch these files when doing a grep in `workdir/*/*.vcf`

Reply:
Do we want to split these out into separate processes?

I suspect you don't want to do that now.. But maybe in the future?

These multi-command-processes have been quite a headache

Reply:
yes. that new info header thingie would be suited well as a reusable util process,